cabal.project

```diff
@@ -0,0 +1,7 @@
+packages:
```

Comment:
This looks redundant now that you released it.

Reply:
Thanks!